go.mod

```diff
@@ -0,0 +1,3 @@
+module gopkg.in/urfave/cli.v2
+
+go 1.12
```

Comment:
Lets leave this out. The team needs to decide whats happening with the different versions.

Reply:
@AudriusButkevicius alright, I will remove the go.mod file for now.